src/status_im/subs/chats.cljs

```diff
@@ -358,3 +358,21 @@
    (re-frame/subscribe [:chats/chat chat-id]))
  :->
  :image)
+
+(re-frame/reg-sub
+ :group-chat/manage-members-count
+ :<- [:chats/chats]
+ :<- [:group-chat/selected-participants]
+ :<- [:group-chat/deselected-members]
+ (fn [[chats selected-participants deselected-members] [_ chat-id]]
+   (let [contacts               (get-in chats [chat-id :contacts])
+         current-contacts-count (count contacts)
```

Comment:
Aggregation subs can greatly benefit from using other subs that compute the counts. This way this `:group-chat/manage-members-count` subscription doesn't need to rely on potentially large collections on every state update of the collections. I'm not sure if we have a sub for the count of selected participants or the count of deselected members, etc, but seems worth adding those as a good practice.

Reply:
I’ve added separate subscriptions for selected, deselected, and current member counts, and I'm using them here.